processor/src/range/mod.rs

```diff
@@ -11,6 +11,24 @@ use request::CycleRangeChecks;
 #[cfg(test)]
 mod tests;
 
+// Range check trace table
+pub const RANGE_CHECK_TRACE_TABLE_WIDTH: usize = 256;
+
+// RANGE CHECKER TRACE TABLE
+// ================================================================================================
+
+/// Range checker trace table.
+///
+/// This component will store 8-bit lookup tables so it won't have to be reconstructed multiple
+/// times. It will also contain the 16-bit length calculated while building the 8-bit tables.
+///
+/// This is intended as internal helper structure and is not intended to be exported as part of the
+/// public API.
+pub struct RangeCheckTraceTable {
+    pub lookups_8bit: [usize; RANGE_CHECK_TRACE_TABLE_WIDTH],
+    pub len: usize,
```

Comment:
nit: is `len` the 16-bit length or is it the length of 8-bit plus 16-bit? (It looks like the latter)

Reply:
Its confusing indeed; updated. Its the combined length of 8 and 16 bits